subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/problems/JsonModelWriter.kt

```diff
@@ -112,6 +112,13 @@ class JsonModelWriter(val writer: Writer) {
                         comma()
                         property("declaringType", firstTypeFrom(trace.trace).name)
                     }
+                    PropertyKind.PropertyUsage -> {
+                        property("kind", trace.kind.name)
+                        comma()
+                        property("name", trace.name)
+                        comma()
+                        property("from", trace.sequence.filterIsInstance<PropertyTrace.Project>().first().path)
```

Comment:
Could make `taskPathFrom()` be more general and support "path-identified" traces, or at least have a parallel `projectPathFrom()`.

Reply:
Fixed! I don't think it's worth a more general function that covers both cases, as for now we only have the two call sites, so I went ahead with `projectPathFrom`.